pod-analyzer/README.md

```diff
@@ -1,109 +1,393 @@
-  helm upgrade --install -n kubesphere-system --create-namespace ks-core  https://charts.kubesphere.io/main/ks-core-1.1.0.tgz --set apiserver.nodePort=30881 --debug --wait
-  ```
+   - Run the Ollama container using the following command:
 
-  **3、Install the tools required for extensions**
+   ```bash
+   docker run -d \
+   -v /home/user/ollama:/root/.ollama \
+   -p 11434:11434 \
+   --name ollama \
+   ollama/ollama
+   ```
 
-  ```
-  curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash -
-  sudo apt-get install -y nodejs
-  npm install --global yarn
-  ```
+**Deploy Ollama to Kubernetes**
 
-- **Set up the runtime environment for LLM**
+1. **Create Deployment File**
 
-  **1、Install Ollama**
+   - Create a file named `ollama-deployment.yaml` and add the following content:
 
-  ```
-  pip install modelscope
-  modelscope download --model=modelscope/ollama-linux --local_dir ./ollama-linux --revision v0.3.10
-  cd ollama-linux
-  sudo chmod 777 ./ollama-modelscope-install.sh
-  ./ollama-modelscope-install.sh
-  ```
+   ```yaml
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: ollama
+   spec:
+     selector:
+       matchLabels:
+         app: ollama
+     template:
+       metadata:
+         labels:
+           app: ollama
+       spec:
+         containers:
+         - name: ollama
+           image: ollama/ollama:latest
+           ports:
+           - name: http
+             containerPort: 11434
+             protocol: TCP
+       tolerations:
+         - key: "node-role.kubernetes.io/master"
+           operator: "Exists"
+           effect: "NoSchedule"
+   ---
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: ollama-svc
+   spec:
+     selector:
+       app: ollama
+     ports:
+     - protocol: TCP
+       port: 11434
+       targetPort: 11434
+   ```
+
+2. **Apply Configuration**
+
+   - Apply the configuration using the following command:
+
+   ```bash
+   kubectl apply -f ollama-deployment.yaml
+   ```
+
+3. **Check Deployment Status**
+
+   - Check the deployment status with these commands:
+
+   ```bash
+   kubectl get deployments
+   kubectl get pods
+   kubectl get services
+   kubectl get pods -o wide
+   ```
+
+   - Reference results:
+
+   ![image1](./image/image6.png)
+
+4. **Execute Command**
+
+   - Enter the Pod and execute the command:
+
+   ```bash
+   kubectl exec -it <pod-name> -- ollama pull qwen2:7b
+   ```
+
+   - Replace `<pod-name>` with the actual Pod name.
+
+5. **Install jq **
+
+   - Need to process JSON data, install `jq`:
+
+   ```bash
+   sudo yum install jq
+   ```
+
+6. **Test：Send API Request**
+
+   - Use `curl` to send a request:
+
+   ```bash
+   curl -L 'http://<node-ip>:11434/api/generate' \
+   -H 'Content-Type: application/json' \
+   -d '{
+     "model": "qwen2:7b",
+     "prompt": "Introduce artificial intelligence",
+     "format": "json",
+     "stream": false
+   }'
+   ```
+
+   - Replace `<node-ip>` with the actual node IP address.
```

Comment:
[suggestion] using pod directly is okay but still prefer to use svc as entry point